_includes/header.html

```diff
@@ -17,4 +17,14 @@
       </ul>
     </nav>
   </div>
+  
+  <!-- Global site tag (gtag.js) - Google Ads: 743722644 -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=AW-743722644"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', 'AW-743722644');
+  </script>
```

Comment:
@rozerosie does this have to go in the header or should it go to the bottom of the footer? or does it not matter any more?